user-operator/src/test/java/io/strimzi/operator/user/operator/KafkaUserQuotasTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.user.operator;
+
+import io.strimzi.test.EmbeddedZooKeeper;
+import io.vertx.core.json.JsonObject;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class KafkaUserQuotasTest {
```

Comment:
Should this be an IT if it runs against Zoo? Or do we call ITs only the tests using Kubernetes?

Reply:
@Frawless What do you think?

Reply:
I think what really matters here is how we call these tests in TO for example.

Reply:
Integrationalized via https://github.com/strimzi/strimzi-kafka-operator/pull/2002/commits/869d10f1c1676b23ea4ed46d7d9035b9c3b3a490